markdown/2017-10-16.md

```diff
@@ -0,0 +1,180 @@
+It's important to point out that *we did not run this event*. We apologise
+that J.P. Morgan did not communicate the location of the event as effectively
+as they could've perhaps done.
+
+Last week's annual Sponsors' Exhibition saw over 500 members meeting the 14
+sponsors present, which is an amazing turnout. Thank you to everyone who
+attended the event!
+
+---
+
+Our upcoming events:
+
+# Consumer Credit Checking for Fun & Profit
+
+<center>
+    ![Netcraft Workshop Header](https://i.imgur.com/jdaLsHV.png)
+</center>
+
+- **When:**  <font color="red">**TONIGHT**</font> 17.30-23.00, 16th October 2017
+- **Where:** RSM G01 & the Union Bar thereafter
+- **Link:** [Facebook](https://www.facebook.com/events/1866631386986389/)
+
+Netcraft is running a workshop where you can learn some of the opportunities
+that exist to make catastrophic mistakes when developing and maintaining a
+web application on the Internet.
+
+What sort of programming error can be that expensive? How much is it worth to
+the lawyers? And how much is a mistake like that worth to the hacker who
+finds it?
+
+Your challenge is that you are looking at a credit agency's web interface
+holding credit information on hundreds of millions of people.
+
+Pizza and drinks will be provided at the event. In order to give an accurate
+idea how many staff Netcraft should bring and how much pizza we should buy,
+please press 'Going' on the Facebook event if you intend to come.
+
+There will a free bar night at the Union Bar after the competition.
+
+# The Welcome Bar Night
+
+<center>
+    ![Welcome Bar Night Header](https://i.imgur.com/tMtZTiv.png)
+</center>
+
+- **When:** 19.00-00.00, 19th October 2017
+- **Where:** Metric
+- **Link:** [Facebook](https://www.facebook.com/events/857222977758027/)
+
+It's time for DoCSoc's first bar night of the year: The Welcome Bar Night!
+
+Come along for free drinks and to say hi to our new freshers or catch up with
+friends - it's sure to be a night you don't want to miss out on.
+
+Drinks tokens will be given upon entry for FREE to all DoCSoc members who
+acquired membership before 18.00 on the day of the event. Strictly 18+ only.
+
+---
+
+# Autumn Elections
+
+<center>
+<h3>**VOTING IS OPEN**. [Please vote](https://vote.union.ic.ac.uk/).</h3>
+</center>
+
+This is an opportunity to make your voice heard within the College and the
+Union.
+
+Every year students vote for at least one Academic Year Representative who
+will receive and collate feedback from their cohort and will pass this
+information to the Departmental Representatives or staff as appropriate.
+Academic Representatives have made long-lasting improvements in many areas
+such as the content of curricula, feedback quality and timeliness, exam
+timetables and formats, online learning resources, and the physical
+facilities available.
+ 
+This year the Union has also newly launched a Wellbeing Representation
+Network, which aims to tackle issues such as stress and pressure, mental
```

Comment:
can we hyperlink "Wellbeing Representation Network" to some union page?

Reply:
Could do, but the main elections page has deets on this already and that's linked a bit later. Would that not be redundant?

Reply:
I think link it anyway